doc/changes/devel.rst

```diff
@@ -83,6 +83,7 @@ Bugs
 - Fix bug with :func:`~mne.viz.plot_raw` where changing ``MNE_BROWSER_BACKEND`` via :func:`~mne.set_config` would have no effect within a Python session (:gh:`12078` by `Santeri Ruuskanen`_)
 - Improve handling of ``method`` argument in the channel interpolation function to support :class:`str` and raise helpful error messages (:gh:`12113` by `Mathieu Scheltienne`_)
 - Fix combination of ``DIN`` event channels into a single synthetic trigger channel ``STI 014`` by the MFF reader of :func:`mne.io.read_raw_egi` (:gh:`12122` by `Mathieu Scheltienne`_)
+- Fix bug with :func:`mne.io.read_raw_eeglab` and :func:`mne.io.read_epochs_eeglab` where automatic fiducial detection would fail for certain files (:gh:`12165` by `Clemens Brunner`_) 
```

Comment:
Wrong namespace https://mne.tools/stable/generated/mne.read_epochs_eeglab.html
```suggestion
- Fix bug with :func:`mne.read_raw_eeglab` and :func:`mne.io.read_epochs_eeglab` where automatic fiducial detection would fail for certain files (:gh:`12165` by `Clemens Brunner`_) 
```

Reply:
But why? `mne.io.read_epochs_eeglab()` is defined? It's also in `mne/io/__init__.py` (and `.pyi`)?

Reply:
You have to link to things where we document them in the public API

Reply:
Can you be more specific? What exactly am I looking for here? Do I need to check `doc/api/something`? Or do you mean a specific `__init__.py`?

Reply:
Easiest way is note the URL and name/title are both `mne.read_epochs_eeglab` not `mne.io.read_epochs_eeglab`:

![image](https://github.com/mne-tools/mne-python/assets/2365790/42cfe97b-faaf-4c89-98be-8adcec31437a)

Harder way is too look at our docs and figure it out from the `.. currentmodule:: mne` that precedes it in the reST

https://github.com/mne-tools/mne-python/blob/fcb59266b7616dd99908c6dac0e2295722b0c567/doc/api/file_io.rst#L23

Reply:
OK, thanks Eric!